yarn-project/end-to-end/src/e2e_state_vars.test.ts

```diff
@@ -288,5 +289,66 @@ describe('e2e_state_vars', () => {
       expect(tx.data.forRollup!.rollupValidationRequests.maxBlockNumber.isSome).toEqual(true);
       expect(tx.data.forRollup!.rollupValidationRequests.maxBlockNumber.value).toEqual(new Fr(expectedMaxBlockNumber));
     });
+
+    it('checks max block number propagated by reading with our SharedMutablePrivateGetter in TestContract accessing AuthContract; before and after a decreasing then increasing delay change', async () => {
+      // Our initial max block number will be 5 because that is the value of INITIAL_DELAY
+      const expectedInitialMaxBlockNumber = (await pxe.getBlockNumber()) + 5;
+
+      // Our SharedMutablePrivateGetter here reads from the SharedMutable authorized storage property in AuthContract
+      const tx = await testContract.methods.test_shared_mutable_private_getter(authContract.address, 2).prove();
+
+      // The validity of our SharedMutable read request is limited to 5 blocks, which is our initial delay.
+      expect(tx.data.forRollup!.rollupValidationRequests.maxBlockNumber.isSome).toEqual(true);
+      expect(tx.data.forRollup!.rollupValidationRequests.maxBlockNumber.value).toEqual(
+        new Fr(expectedInitialMaxBlockNumber),
+      );
+
+      // We change the SharedMutable authorized delay here to 100, this means that a change to the "authorized" value can
+      // only be applied 100 blocks after it is initiated, and thus read requests on a historical state without an initiated change is
+      // valid for at least 100 blocks
+      await authContract.methods.set_authorized_delay(2).send().wait();
```

Comment:
The comment is saying 100, but the code is saying 2 👀 

You are first shortening the delay, and then extending it? Is there a special case when we do both of these, or would it be the same if two tests? One that is extending and one reducing?